src/test/java/uk/gov/companieshouse/api/accounts/controller/CurrentPeriodControllerTest.java

```diff
@@ -7,6 +7,7 @@
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.when;
 
+import java.util.HashMap;
 import javax.servlet.http.HttpServletRequest;
 import org.junit.jupiter.api.BeforeEach;
```

Comment:
Could you remove the import for the `BeforeEach` annotation as there are now no other instances of this in the file?